webapi/src/test/scala/org/knora/webapi/TestContainersAll.scala

```diff
@@ -36,18 +36,21 @@ object TestContainersAll {
   val SipiImageName: DockerImageName = DockerImageName.parse(s"daschswiss/knora-sipi:${BuildInfo.version}")
   val SipiContainer = new GenericContainer(SipiImageName)
   SipiContainer.withExposedPorts(1024)
+  SipiContainer.withEnv("KNORA_WEBAPI_KNORA_API_EXTERNAL_HOST", "0.0.0.0")
+  SipiContainer.withEnv("KNORA_WEBAPI_KNORA_API_EXTERNAL_PORT", "3333")
   SipiContainer.withEnv("SIPI_EXTERNAL_PROTOCOL", "http")
   SipiContainer.withEnv("SIPI_EXTERNAL_HOSTNAME", "0.0.0.0")
   SipiContainer.withEnv("SIPI_EXTERNAL_PORT", "1024")
   SipiContainer.withEnv("SIPI_WEBAPI_HOSTNAME", localIpAddress)
   SipiContainer.withEnv("SIPI_WEBAPI_PORT", "3333")
-  SipiContainer.withCommand("--config=/sipi/config/sipi.knora-docker-config.lua")
+
+  SipiContainer.withCommand("--config=/sipi/config/sipi.docker-config.lua")
 
   // TODO: Needs https://github.com/scalameta/metals/issues/3623 to be resolved
   SipiContainer.withClasspathResourceMapping(
-    // "/sipi/config/sipi.knora-docker-config.lua"
-    "/sipi.knora-docker-config.lua",
-    "/sipi/config/sipi.knora-docker-config.lua",
+    // "/sipi/config/sipi.docker-config.lua"
```

Comment:
are those comments still valid and needed?

Reply:
yes, they belong to the TODO which is still open (we have to wait for a fix in the metals plugin)